candidates/tests/test_mark_change_as_reviewed_view.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django_webtest import WebTest
+
+from .auth import TestUserMixin
+from ..models import ChangeReviewed, LoggedAction
+from candidates.tests import factories
+
+from popolo.models import Person
+
+
+class TestMarkChangeAsReviewedView(TestUserMixin, WebTest):
+    def setUp(self):
+        test_person_1 = factories.PersonExtraFactory.create(
+            base__id=9876,
+            base__name='Test Candidate for Recent Changes',
+        )
+        self.action1 = LoggedAction.objects.create(
+            user=self.user,
+            action_type='person-create',
+            ip_address='127.0.0.1',
+            person=test_person_1.base,
+            popit_person_new_version='1234567890abcdef',
+            source='Just for tests...',
+        )
+        self.app.get(
+            '/recent-changes',
+            user=self.user,
+        )
+
+    def tearDown(self):
```

Comment:
This isn't obvious at all, but you actually don't need the `tearDown` - each test method is run in a transaction which is rolled back afterwards, and the `setUp` method is run within that transaction.